pynestml/cocos/co_co_all_variables_defined.py

```diff
@@ -76,8 +76,8 @@ def check_co_co(cls, node):
                  and symbol.block_type != BlockType.INPUT_BUFFER_SPIKE \
                  and not symbol.get_referenced_object().get_source_position().is_added_source_position():
                     # except for parameters, those can be defined after
-                    if (not symbol.get_referenced_object().get_source_position().before(var.get_source_position()) and
-                            symbol.block_type != BlockType.PARAMETERS):
+                    if ((not symbol.get_referenced_object().get_source_position().before(var.get_source_position()))
+                     and (not symbol.block_type in [BlockType.PARAMETERS, BlockType.INTERNALS])):
```

Comment:
isn't this indentation a bit weird?
Does this also fix #510?

Reply:
Are you calling my personal indentation style weird? :)

I think your suggestion of a (partial) PEP8 enforcement would definitely be good, but I would propose creating a separate issue for that, because it's not conceptually related to this PR.

You're right, this fixes #510 also!

Reply:
> Are you calling my personal indentation style weird? :)

;P

> I think your suggestion of a (partial) PEP8 enforcement would definitely be good, but I would propose creating a separate issue for that, because it's not conceptually related to this PR.

Agreed

> You're right, this fixes #510 also!

Great!